src/org/zaproxy/zap/extension/pscan/DialogModifyAutoTagScanner.java

```diff
@@ -56,10 +56,6 @@ protected boolean validateName(String name) {
     @Override
     protected void init() {
         getNameTextField().setText(scanner.getName());
-        getRequestUrlRegexTextField().discardAllEdits();
```

Comment:
It should call `getNameTextField().discardAllEdits();` instead.

Reply:
fixd